README.md

```diff
@@ -1,14 +1,13 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2122S2-CS2103T-W09-4/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S2-CS2103T-W09-4/tp/actions)
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
+* **TAPA** allows TAs to better manage their student’s contact and progress, especially for those who are teaching multiple classes/modules at the same time. It is optimised for use on CLI.<br>
+  Usage:
+  * Check out **[TAPA's User Guide](https://github.com/AY2122S2-CS2103T-W09-4/tp/blob/master/docs/UserGuide.md)**.
+* The project is a software project for a desktop application that manages student contact details and progress.
+  * It is **written in OOP fashion**.
   * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+* It is named `Teaching Assistant Personal Assistant` (`TAPA` for short) because it helps TAs manage their students' contacts and progress.
```

Comment:
```suggestion
* It is named `Teaching Assistant's Personal Assistant` (`TAPA` for short) because it helps TAs manage their students' contacts and progress.
```

Reply:
Alright, will edit that, thanks!